pageserver/src/tenant.rs

```diff
@@ -3627,7 +3627,7 @@ impl Tenant {
         start_lsn: Lsn,
         ancestor: Option<Arc<Timeline>>,
         last_aux_file_policy: Option<AuxFilePolicy>,
-    ) -> anyhow::Result<UninitializedTimeline> {
+    ) -> anyhow::Result<UninitializedTimeline<'a>> {
```

Comment:
I think the proposed fix by clippy is to remove `'a` from the signature?

Reply:
...instead of adding `'a` here

Reply:
No, the `'a` lifetime is needed for the other arguments